### PR TITLE
SW-5331 Show loading page when determining module projects

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2607?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2607** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2607?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2606** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2606?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/providers/Participant/ParticipantProvider.tsx [hunk 1]
```diff
@@ -49,16 +49,18 @@ const ParticipantProvider = ({ children }: Props) => {
     activeModules,
     moduleProjects,
     modules,
+    orgHasModules: orgHasModules,
+    orgHasParticipants: orgHasParticipants,
```
Comment (nickgraz) [inline src/providers/Participant/ParticipantProvider.tsx:53]:
```suggestion
    orgHasModules,
    orgHasParticipants,
```

**Review by nickgraz — APPROVED** [review]
Looks good to me, one small suggestion

**Comment by tommylau523** [comment]
### Merge activity

* **May 13, 6:44 PM EDT**: @tommylau523 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2607).

**Pull request merged**